longChatPane.js

```diff
@@ -258,15 +266,15 @@ module.exports = {
 
     div.setAttribute('class', 'chatPane')
     const options = { infinite: true } //  was: menuHandler: menuHandler
-    const context = { noun: 'chat room', div, dom }
-    context.me = UI.authn.currentUser() // If already logged on
+    const context3 = { noun: 'chat room', div, dom: dom }
```

Comment:
Hmm, the `3` here is not really informative... Maybe change it to a name that describes what kind of context this is? (If you know - I have no idea :/)

Reply:
Not really informative, but context is a popular name in some cases =P

But I saw that we didn't need variables for two of the objects, as they are only used one and we can simply inline them. So hopefully easier to read now.